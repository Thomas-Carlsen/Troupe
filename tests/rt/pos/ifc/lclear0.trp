(* positive test; should be ok *)
let val c = raisembox ({alice})    
    val () = lowermbox (c, authority)
in c
end 