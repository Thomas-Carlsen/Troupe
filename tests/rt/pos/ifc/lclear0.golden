2020-02-09T01:34:48.124Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: "64b0b9ee-3837-4d68-ae4e-dfaa21394315"@{}%{}
